src/main/java/io/github/genomicdatainfrastructure/daam/services/AttachFileToApplicationService.java

```diff
@@ -48,4 +49,15 @@ public AddedAttachment attach(Long applicationId, File file, String userId) {
                 .id(attachmentResponse.getId())
                 .build();
     }
+
+    private File tempFile(String filename, File file) {
```

Comment:
on docker based system, i would expect enough diskspace is given during container generation for this to not fail

Reply:
Very good concern. I'm assuming that as well.